src/main/java/org/humancellatlas/ingest/user/UserController.java

```diff
@@ -106,6 +110,13 @@ private Account getCurrentAccount() {
         return account;
     }
 
+    @GetMapping("/list")
+    ResponseEntity<List<Account>> listUsers(
+            @RequestParam("role") Optional<Role> role) {
+        return role.map(value -> ResponseEntity.ok(accountRepository.findAccountByRolesEquals(value))).orElseGet(() -> ResponseEntity.ok(accountRepository.findAll()));
+    }
+
+
```

Comment:
I think the endpoint should be just to get the list of wranglers. And only wranglers should have permission to do it.

Reply:
I think it's good to have a general endpoint that can also be used to fetch users of other roles, if ever needed in the future.

How do I limit permissions of the api endpoint?

Reply:
There is a similar code in the AuthenticationController on checking the role of the user who sent the request. 
```
@GetMapping(path="/account", produces=APPLICATION_JSON_UTF8_VALUE)
    public ResponseEntity<Account> getAccount(Authentication authentication) {
        if (authentication.getAuthorities().contains(Role.GUEST)) {
            return ResponseEntity.notFound().build();
        }
        Account account = (Account) authentication.getPrincipal();
        return ResponseEntity.ok().body(account);
    }

```
I'm still not very comfortable with having a public endpoint returning all users info. However, I think there's a larger work needed to design and implement authorisation properly in the core API 🤔   I also noticed there are 2 classes account & user referring to the same entity. That needs to be fixed. But maybe not in this ticket.

Reply:
Hmm, I'll approve it now, as we don't really store a lot of info in the Account entity. But if in case we have to add more info, e.g. email addresses or other we might want to revisit the access to this. If you have more time for this sprint, could we at least just allow users only with wrangler's role to access this endpoint?

Reply:
What about doing this via the `SecurityConfig`?

Reply:
How about making use of the `SecurityConfig` here?